packages/extension-ui/src/Popup/Accounts/Account.tsx

```diff
@@ -54,12 +56,14 @@ function Account({
   }, [isSelected]);
 
   const _onCheckboxChange = useCallback(() => {
+    checkBoxOnChange && checkBoxOnChange(true);
+
```

Comment:
Let's try not to use `&&` if we don't assign this to a const or is not in JSX template
`if` was created for a reason :)